package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "dd-trace",
-  "version": "6.0.0-pre",
+  "version": "5.57.0",
```

```suggestion
  "version": "6.0.0-pre",
```

todo: revert after successful ci run